draft-pauly-ohai-svcb-config.md

```diff
@@ -214,20 +234,31 @@ There are several approaches clients can use to mitigate key targetting
 attacks. {{?CONSISTENCY=I-D.draft-wood-key-consistency}} provides an analysis
 of the options for ensuring the key configurations are consistent between
 different clients. Clients SHOULD employ some technique to mitigate key
-targetting attack. One mitigation specific to this mechanism is validating
-that SVCB or HTTPS records including the "oblivious-configs"
-are protected by DNSSEC {{?DNSSEC=RFC4033}}. This prevents attacks
-where a unique response is generated for each client of a resolver.
+targetting attack.
```

Comment:
I might add some text here about how the SVCB parameter is just a hint, and clients _could_ choose to query every target for their oblivious configs if they wanted to. In that way, DNSSEC isn't required -- it's just an optimization.

Reply:
Added text about being a hint earlier in the security section.